pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1380,6 +1395,11 @@ func parseIpspec(ipspec *zconfig.Ipspec,
 		}
 		config.DhcpRange.Start = start
 		config.DhcpRange.End = end
+	} else if config.Subnet.IP != nil {
+		// set default DhcpRange
+		log.Warnf("No Dhcp configuration, setting the default\n")
```

Comment:
Ditto

Reply:
And no trailing \n in any new log messages.

Reply:
done